src/api/app/models/token/workflow.rb

```diff
@@ -10,23 +10,35 @@ def call(options)
     return unless extractor.allowed_event_and_action?
 
     scm_extractor_payload = extractor.call
-    yaml_file = Workflows::YAMLDownloader.new(scm_extractor_payload).call
+    yaml_downloader = Workflows::YAMLDownloader.new(scm_extractor_payload)
```

Comment:
This is undoing changes from #11338. Please revert your changes.

Reply:
Ouch, I messed up the rebase.